### PR TITLE
tests(Wallet): add tests for new seed phrase corner cases

### PR DESCRIPTION
**Pull request by stefandunca**
## Regression tests for the issue fixed by https://github.com/status-im/status-desktop/pull/7885

~~Cover the copy-paste corner case for specific seed phrases when having three-words~~
Following the recent decisions to keep tests running time in a practical time frame (under 3 minutes for wallet) I removed these new regression tests and only kept the other test framework improvements.

Also, change the seed phrase filling used in the test to a more realistic method by avoiding the direct input that doesn't generate the event. This way we cover another corner case that relies on user input changing focus and was just fixed

Fix the animation of seed phrase input affecting the clicking using the new seed-phrase inserting.

Updates: #7715

### CODE REVIEW

**Review by saledjenic — APPROVED** [review]

**Review by noeliaSD — CHANGES_REQUESTED** [review]
Code looks good to me but I have issues executing the tests (and once they are correct, please remove the tag `mayfail` if the scenarios are reliable and not broken):

- Both, `User imports a seed phrase using copy-paste` and  `User imports a seed phrase`cannot finish the  last step because the `Add` button is disabled. See screenshot:

Screenshot 2022-10-14 at 13 34 10

File: test/ui-test/src/common/SeedUtils.py [hunk 1]
```diff
@@ -2,34 +2,33 @@
 from drivers.SquishDriver import *
 from drivers.SquishDriverVerification import *
 
+# Simulate user input of seed phrase by using mouse to trigger enabled events
```
[the remark this answers is not in the document]
Comment (stefandunca) [reply test/ui-test/src/common/SeedUtils.py:5]: > please remove the tag mayfail if the scenarios are reliable and not broken

Will do, thanks for the heads up

**Review by noeliaSD — APPROVED** [review]
